__tests__/app.test.js

```diff
@@ -110,4 +110,40 @@ describe("/api/articles/:atricle_id", () => {
       });
     });
   });
+  describe("/api/articles/:article_id/comments", () => {
+    test("200 OK: should return an array of comments refferencing the article_id that was given", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          console.log(body);
+          if (body.comments.length > 0) {
+            body.comments.forEach((comment) => {
+              expect(typeof comment.comment_id).toBe("number");
+              expect(typeof comment.votes).toBe("number");
+              expect(typeof comment.created_at).toBe("string");
+              expect(typeof comment.author).toBe("string");
+              expect(typeof comment.body).toBe("string");
+              expect(typeof comment.article_id).toBe("number");
+            });
+          }
+        });
+    });
```

Comment:
could do with a test and behaviour for this 
- Status 200, valid ID, but has no comments responds with an empty array of comments